.doc_gen/readmes/README.md

```diff
@@ -0,0 +1,103 @@
+# WRITEME: auto-generated READMEs
+
+## Overview
+
+Describes WRITEME, a tool to automatically generate service-level READMEs from
+metadata and Jinja templates.
+
+This is an internal tool intended for use only by the AWS code examples team.
+
+## Prerequisites
+
+You must have a recent version of Python installed to run this tool, 
+and a recent version of pip (Python's package manager) to install the
+required packages.
+
+### Install packages
+
+A virtual environment is recommended. Create a virtual environment 
```

Comment:
A virtual environment is recommended. --> We recommend a virtual environment.

Reply:
fixed